### PR TITLE
Buildout ui

### PR DESCRIPTION
**Pull request by allthesignals**
This PR does the following: 
33340a6d Add school leaflet map
b2876677 Update routes
337f90c1 Add district searches UI
710f7932 Fix active survey query
62bfd6f9 Paginate districts; Override kaminari templates
1039593c Scaffold districts, schools for list views
0662dc04 Remove turbolinks; Refactor menu and splash
d6a371be Setup test spec and some fixtures
96a66c54 Finalize bin/setup

### CODE REVIEW

File: logfile [hunk 1]
```diff
@@ -561,3 +561,25 @@ LOG:  checkpoints are occurring too frequently (17 seconds apart)
...
+ERROR:  database "mysc-test" already exists
+STATEMENT:  CREATE DATABASE "mysc-test" ENCODING = 'utf8'
+LOG:  checkpoints are occurring too frequently (19 seconds apart)
+HINT:  Consider increasing the configuration parameter "max_wal_size".
+LOG:  checkpoints are occurring too frequently (15 seconds apart)
+HINT:  Consider increasing the configuration parameter "max_wal_size".
+LOG:  received smart shutdown request
+LOG:  autovacuum launcher shutting down
+LOG:  shutting down
+LOG:  database system is shut down
```
Comment (mzagaja) [inline logfile]: Should this file be in .gitignore?

Comment (allthesignals) [inline logfile]: Yes

**Comment by allthesignals** [comment]
@mzagaja please QA tomorrow when you get a chance. Maybe a more specific ask: test if it's possible to `bin/setup` the app. 

As for test suite, there's not one really to speak of. Was hoping to get some guidance on where specifically to start, get some examples down. :)

**Comment by mzagaja** [comment]
Ok /bin/setup failed on me with the following errror:

```
== Import CSV Seed Data ==
rails aborted!
ActiveRecord::StatementInvalid: PG::UndefinedTable: ERROR:  relation "districts" does not exist
LINE 8:                WHERE a.attrelid = '"districts"'::regclass
                                          ^
:               SELECT a.attname, format_type(a.atttypid, a.atttypmod),
                     pg_get_expr(d.adbin, d.adrelid), a.attnotnull, a.atttypid, a.atttypmod,
             (SELECT c.collname FROM pg_collation c, pg_type t
               WHERE c.oid = a.attcollation AND t.oid = a.atttypid AND a.attcollation <> t.typcollation),
                     col_description(a.attrelid, a.attnum) AS comment
                FROM pg_attribute a LEFT JOIN pg_attrdef d
                  ON a.attrelid = d.adrelid AND a.attnum = d.adnum
               WHERE a.attrelid = '"districts"'::regclass
                 AND a.attnum > 0 AND NOT a.attisdropped
               ORDER BY a.attnum
```

I think it's worth trying to use pg_dump to export in Postgres's custom format:

`pg_dump -h db.live.mapc.org -Fc -o -U editor development_database > massbuilds-04-24-2017.dump`

**Pull request merged**